### PR TITLE
add my homework

### PR DESCRIPTION
**Pull request by inna-shutka**
✅ I did my homework.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+            <form class="form__form-section form-section form--second">
+                <span class="form-section__job">
+                    <span class="job">
+                        <label class="form__label label label--manager">Your Manager</label>
+                        <input class="form__input input input--manager"
+                            type="text"
+                            placeholder=""
+                            required
+                            minlength="2"
+                            maxlength="30">
```
Comment (a1exalexander) [inline index.html]: `readonly` or `disabled`
image

**Review by a1exalexander — APPROVED** [review]
Perfect! But fix one thing with input

You can merge PR after the fix

File: index.html (hunk 1 above)
Comment (a1exalexander) [reply index.html]: you can remove modification class `input--manager` and add `:disabled` or `:read-only` styles

Comment (inna-shutka) [reply index.html]: 🐛Fix it!

**Pull request merged**